cfme/rest/gen_data.py

```diff
@@ -368,25 +368,34 @@ def automation_requests_data(vm, requests_collection=False, approve=True, num=4)
     return [data for _ in range(num)]
 
 
-def groups(request, appliance, role, tenant, num=1):
+def groups(request, appliance, role, tenant, num=1, **kwargs):
     data = []
     for _ in range(num):
-        data.append({
-            "description": "group_description_{}".format(fauxfactory.gen_alphanumeric()),
-            "role": {"href": role.href},
-            "tenant": {"href": tenant.href}
-        })
+        data.append(
+            {
```

Comment:
This indentation style is rare to do in Python.

Reply:
I formatted the code with black and this is how it was formatted.